src/slash_command.ts

```diff
@@ -3,126 +3,87 @@ import {TerraformCli} from "./tfc_cli";
 import {TerraformCloudApi} from "./tfc_api";
```

Comment:
It may be easier to understand the changes to this file by looking at the file in its current state on the branch rather than looking at the diff.

I didn't really change much here. I moved code around to better isolate concerns so that the logic flow of switching backends would be easier to follow.